tiledb/sm/array_schema/array_schema.h

```diff
@@ -574,6 +588,11 @@ class ArraySchema {
   /*         PRIVATE ATTRIBUTES        */
   /* ********************************* */
 
+  /**
+   * The memory tracker of the array this fragment metadata corresponds to.
```

Comment:
`s/fragment metadata/ArraySchema/`

Reply:
Unfortunately I have no idea what this comment means :(

Reply:
Ah, sorry. Showing my age with sed syntax. The comment here says "fragment metadata" when it should say "ArraySchema".